.gitignore

```diff
@@ -21,3 +21,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+__snapshots__
```

Comment:
Are you sure you don't need the snapshots in the repo? Without snapshots your tests won't work if you clone the repo.